### PR TITLE
[11.0][IMP] sale_order_secondary_unit: Add security group

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
This PR depends of https://github.com/OCA/product-attribute/pull/445

### CODE REVIEW

File: sale_order_secondary_unit/migrations/11.0.1.0.2/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    # Remove old security group_uom from product template view because now has
+    # a new group for secondary units
+    group_uom = env.ref('product.group_uom')
+    view_product_template = env.ref(
+        'sale_order_secondary_unit.product_template_form_view')
+    view_product_template.write({
+        'groups_id': [(3, group_uom.id)]
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/migrations/11.0.1.0.2/post-migration.py:14]: Isn't enough with putting in the XML `groups_id` empty?

Comment (sergio-teruel) [reply sale_order_secondary_unit/migrations/11.0.1.0.2/post-migration.py:14]: I had two options, one is puting in the xml groups_id = [(5,0), (4, ref('new_group'))] and the other in a post_migration remove the old security group. I have chosen the second.

Comment (pedrobaeza) [reply sale_order_secondary_unit/migrations/11.0.1.0.2/post-migration.py:14]: But this technique on views seems a bit weird, and most being noupdate=0. Even more, `groups_id` in views doesn't use `(4,` terminology if I don't remember bad, but directly use the ref of the group.

**Comment by sergio-teruel** [comment]
How can I restart the CI?

**Comment by pedrobaeza** [comment]
I have forced a rebuild on runbot

File: sale_order_secondary_unit/migrations/11.0.1.0.2/post-migration.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_order_secondary_unit/migrations/11.0.1.0.2/post-migration.py:14]: Let's continue this way as I'm not sure.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**